docs/src/intro-java.md

```diff
@@ -1,150 +1,96 @@
 
-<Tabs
-  defaultValue="java"
-  values={[
-    {label: 'Example.java', value: 'java'},
-    {label: 'pom.xml', value: 'pom'}
-  ]
-}>
-<TabItem value="java">
+Create a new [Maven](https://maven.apache.org/guides/getting-started/maven-in-five-minutes.html) project:
 
-```java
-package org.example;
+```bash
+mvn archetype:generate -DgroupId=com.mycompany.app -DartifactId=my-app -DarchetypeArtifactId=maven-archetype-quickstart -DarchetypeVersion=1.4 -DinteractiveMode=false
+```
 
-import com.microsoft.playwright.*;
+Add the Playwright dependency in the `pom.xml` file:
 
-public class Example {
-  public static void main(String[] args) {
-    try (Playwright playwright = Playwright.create()) {
-      Browser browser = playwright.chromium().launch();
-      Page page = browser.newPage();
-      page.navigate("http://playwright.dev");
-      System.out.println(page.title());
-    }
-  }
-}
+```xml
+<dependency>
+    <groupId>com.microsoft.playwright</groupId>
+    <artifactId>playwright</artifactId>
+    <version>1.27.0</version>
+</dependency>
 ```
 
-</TabItem>
-<TabItem value="pom">
+And set the `<maven.compiler.source>` and `<maven.compiler.target>` to `1.8` in the `pom.xml` file:
 
 ```xml
-<?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-
-  <groupId>org.example</groupId>
-  <artifactId>examples</artifactId>
-  <version>0.1-SNAPSHOT</version>
-  <name>Playwright Client Examples</name>
-  <properties>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-  </properties>
-  <dependencies>
-    <dependency>
-      <groupId>com.microsoft.playwright</groupId>
-      <artifactId>playwright</artifactId>
-      <version>1.27.0</version>
-    </dependency>
-  </dependencies>
-  <build>
-    <plugins>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-compiler-plugin</artifactId>
-        <version>3.1</version>
-        <configuration>
-          <source>1.8</source>
-          <target>1.8</target>
-        </configuration>
-      </plugin>
-    </plugins>
-  </build>
-</project>
+<properties>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+</properties>
 ```
 
-</TabItem>
-</Tabs>
-
-With the Example.java and pom.xml above, compile and execute your new program as follows:
+Install the dependencies:
 
 ```bash
-mvn compile exec:java -Dexec.mainClass="org.example.Example"
+mvn install
 ```
 
-Running it downloads the Playwright package and installs browser binaries for Chromium, Firefox and WebKit. To modify this behavior see [installation parameters](./browsers.md#installing-browsers).
-
-## First script
+## Add Example Test
```

Comment:
It's a bit odd that we call it a test while it doesn't use any test runners (JUnit, TestNG etc.). Maybe we need a doc about integration with the testing frameworks.